src/app.py

```diff
@@ -74,6 +93,32 @@
     ])
 ])
 
+# Define callback to update line chart based on dropdown selection
+@app.callback(
+    Output('crime-line-chart', 'figure'),
+    [Input('crime-type-dropdown', 'value'),
+     Input('neighbourhood-dropdown','value')]
+)
+def update_line_chart(selected_crime,selected_neighbourhood):
+
```

Comment:
Do we need docstring?

Reply:
I dont remember it being a hard requirement. Do you think we can implement it next milesstone?